<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidgetLeft">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>10</y>
      <width>437</width>
      <height>508</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidgetRight">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>436</width>
      <height>508</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveDepths">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Depths</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="mnuOpenPTX"/>
    <addaction name="mnuLoadMask"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mnuOpenPTX">
   <property name="text">
    <string>Open PTX</string>
   </property>
  </action>
  <action name="mnuLoadMask">
   <property name="text">
    <string>Load Mask</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
